js-src/src/app/map.component.html

```diff
@@ -2,7 +2,10 @@
     <div class="col-sm-3">
         <h4><span class="tag tag-default">Score</span> {{currentScore.toFixed(2)}}</h4>
     </div>
-    <div class="col-sm-6" style="text-align: center;">
+    <div class="col-sm-3 countdown">
```

Comment:
looks good! I've tried to replace the city title with the count down, which would look better, but then its hiding the city name which is odd - maybe we can put the countdown instead of the button, when we have the workflow down

```
<div class="col-sm-6 city-label">
        <h2 [hidden]="!countDown" class="countdown">Next city in {{countDown}}..</h2>
        <h2 [hidden]="countDown" *ngIf="currentRound >= 0">{{game.cities[currentRound].name}}</h2>
</div>
```